### DISCUSSION TITLE
Debug workflow using PV/PVC

### DISCUSSION THREAD

**Discussion by xmnlab** [argoproj/argo-workflows#5120] [Q&A]
Hi everyone!

I am trying to use PV/PVC for a simple example, but it is not working.

without the `volume` information, the pod is executed and it is finished in just few seconds.

with the `volume` information it is "running forever" (I waited for 11h and so I decided to terminate the process) with the follow state:

```
    State:          Waiting
      Reason:       ContainerCreating
```

I have tried a lot of things but with no success, also I tried the steps here https://github.com/kubernetes-csi/csi-driver-smb/blob/master/docs/csi-debug.md

I have here some configuration I am using and a log: https://gist.github.com/xmnlab/6b18a00e83e568ba3eb5a84cbc08d66d

the log stops at `Workflow event` and if I try to get the status of the job, the result is:

```
argo get -n argo @latest
Name:                titanic-wf65cmv
Namespace:           argo
ServiceAccount:      default
Status:              Running
Created:             Tue Feb 16 10:25:14 -0400 (8 minutes ago)
Started:             Tue Feb 16 10:25:14 -0400 (8 minutes ago)
Duration:            8 minutes 24 seconds
Progress:            0/1
STEP                      TEMPLATE   PODNAME                     DURATION  MESSAGE
 ● titanic-wf65cmv        main                                                                
 └───◷ generate-artifact  load-data  titanic-wf65cmv-3525859734  8m        ContainerCreating  
```

any ideas about what could be the problem or how can I debug this "infinite" `ContainerCreating` state? thanks

PS: this example duration is just 8 minutes, but I had one before with more than 11 hours

**Comment by alexec** [comment]
I'm not sure - @sarabala1979 - thoughts?

**Comment by xmnlab** [comment]
PS: I am using minikube. I will investigate if it could be related to that.
